src/compression_chunk_size.h

```diff
@@ -21,5 +21,6 @@ typedef struct TotalSizes
 } TotalSizes;
 
 extern TSDLLEXPORT TotalSizes ts_compression_chunk_size_totals(void);
+extern TSDLLEXPORT int64 ts_get_before_compression_row_count(int32 uncompressed_chunk_id);
```

Comment:
This function is still not consistent with the other two public functions in this module.

Reply:
renamed.